src/manual/task-execution/workers.md

```diff
@@ -0,0 +1,61 @@
+---
+filename: task-execution/workers.md
+title: Workers
+order: 20
+---
+
+Workers are the component responsible for actually executing tasks. Taskcluster
+operates on the "pull" model, where workers pull tasks from queues as they have
+available capacity.
+
+Typically workers are implemented as daemons running on compute instances in
+the cloud. As such, they have no REST API -- that is, there is no "worker
+service".
+
+## Worker Groups and Ids
+
+Each individual worker is identified by a `<workerGroup>/<workerId>` pair.
+This information can be useful to identify errant workers.
+
+## Claiming Tasks
+
+Workers signal their availability for new work by calling the Queue's
+`claimWork` method. If there is a task available, this method returns its
+details and records a "claim" of that task by the calling worker. The claim
+lasts for a short time - a matter of minutes - and must be renewed
+(`reclaimTask`) before it expires. If a claim does expire, the task run is
+marked as `exception` and the task may be automatically retried.
```

Comment:
If a claim does expire, it implies a the Queue-worker communication has broken down, and so the Queue will resolve the task run as exception / claim-expired, and schedule another task run so that another worker may claim the task. This will happen up to a maximum number of reclaims.